README.adoc

```diff
@@ -106,33 +100,15 @@ Adding this dependency allows you to use the fault tolerance policies in your mi
 You can also find the `mpFaultTolerance-1.0` feature in your `src/main/liberty/config/server.xml` server configuration,
 which turns on MicroProfile Fault Tolerance capabilities in Open Liberty.
 
-To easily work through this guide, the two provided microservices are set up to run
-on the same server. To simulate the availability of the services and then to enable fault tolerance,
-dynamic configuration with MicroProfile Configuration is used so that you can easily take one service
-or the other down for maintenance. If you want to learn more about setting up dynamic configuration,
-see https://openliberty.io/guides/microprofile-config.html[Configuring microservices].
-
-The following two steps set up the dynamic configuration on the `system` service and its client.
-You can move on to the next section, which adds the fallback mechanism on the `inventory` service.
-
-First, the `src/main/java/io/openliberty/guides/system/SystemResource.java` class has
-the `inMaintenance()` condition, which determines that the system properties are returned only if you
-set the `io_openliberty_guides_system_inMaintenance` configuration property
-to `false` in the `resources/CustomConfigSource.json` file.
-Otherwise, the service returns a `Status.SERVICE_UNAVAILABLE` message, which makes it unavailable.
+To demonstrate the `@Fallback` mechanism, the microservice simulates the `system` service is in maintenance.
```

Comment:
Please consider if we should expand here.  Or, at least, revise the sentence ... What is "the microservice"?  Are we skipping too much?

Reply:
The "microservice" is described in the intro section. It may be good to see the merged version and if additional expansion is needed.

Reply:
This "the microservice" shows up for the first time here in this section.  It still begs the question what is this "the microservice".  IMO, we should revise and clarify what we are trying to convey here.